### PR TITLE
Add get /metrics to nonResourceURLs rbac

### PR DESCRIPTION
**Pull request by hkaj**
#### What this PR does / why we need it:

Add `/metrics` to the bare endpoints the agent can access.

This is required to support querying endpoints protected by RBAC, by [kube-rbac-proxy](https://github.com/brancz/kube-rbac-proxy/) for instance.

(mirrors https://github.com/DataDog/datadog-agent/pull/3178)

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by hkaj** [comment]
bump - will wait for other contributors PRs to get merged before merging this one.

**Review by clamoriniere — APPROVED** [review]

**Pull request merged**
